automation/JJB/ruby_analysis.yaml

```diff
@@ -9,11 +9,11 @@
     parameters:
       - string:
           name: SONAR_KEY
-          default: sonarqube_foreman_ruby_full_analysis
+          default: sonarqube_ruby_full_analysis
```

Comment:
Slight nit: I recommend a consistant naming scheme, so this would be "sonarqube_ruby_analysis".